runtime/vm/jvminit.c

```diff
@@ -5764,6 +5764,27 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 		Trc_VM_contendedLinesizeFailed(queryResult);
 	}
 
+	/* check processor support for cache writeback */
+#if defined(J9X86) || defined(J9HAMMER)
+	J9ProcessorDesc desc;
+	j9sysinfo_get_processor_description(&desc);
+	/* cache line size in bytes is the value of bits 8-15 * 8 */
+	U_32 lineSize = ((desc.features[2] & 0xFF00) >> 8) * 8;
+	vm->cpuCacheLineSize = lineSize;
```

Comment:
Do we not already have this in the VM for contended?

Reply:
Was not sure if they referred to the same cache but it looks like it.